docs/game_sdk/Activities.md

```diff
@@ -95,7 +95,7 @@ For more detailed information and documentation around the Rich Presence feature
 | Join     | 1     |
 | Spectate | 2     |
 
-## Register
+## RegisterCommand
 
 Registers a command by which Discord can launch your game. This might be a custom protocol, like `my-awesome-game://`, or a path to an executable. It also supports any lauch parameters that may be needed, like `game.exe --full-screen --no-hax`.
```

Comment:
I'm not sure it's necessary but clarification couldn't hurt. I'd imagine terminal games (??) would be the only one-offs that aren't bundled, right?

Reply:
@adill this breaks if you run in Unity. It throws an error. But this is a problem since you can't register a game to test with where before it fellback just to the file. Could we maybe just make it not error? and instead log a warning?

Reply:
It looks like the behavior is the same here as it was in the rich presence sdk. If a command string is passed we use the json file, if not then we use launch services. I'm actually not clear on whether that is desired / correct.

That said -- I think making this function warn on failure and not return a `DiscordError` would be fine. This is one of those things that you can't really take action on and it is totally imprudent to `exit(1)` if you get an error.